package.json

```diff
@@ -54,28 +53,28 @@
   "devDependencies": {
     "@commitlint/cli": "^11.0.0",
     "@commitlint/config-conventional": "^11.0.0",
-    "@microsoft/api-extractor": "^7.12.0",
+    "@microsoft/api-extractor": "^7.13.0",
     "@rollup/plugin-alias": "^3.1.1",
     "@rollup/plugin-commonjs": "^16.0.0",
-    "@rollup/plugin-node-resolve": "^11.0.0",
+    "@rollup/plugin-node-resolve": "^11.1.0",
     "@rollup/plugin-replace": "^2.3.4",
     "@types/jest": "^26.0.19",
-    "@typescript-eslint/eslint-plugin": "^4.11.0",
-    "@typescript-eslint/parser": "^4.12.0",
+    "@typescript-eslint/eslint-plugin": "^4.14.0",
+    "@typescript-eslint/parser": "^4.14.0",
     "chalk": "^4.1.0",
-    "eslint": "^7.16.0",
-    "eslint-config-prettier": "^7.1.0",
+    "eslint": "^7.18.0",
+    "eslint-config-prettier": "^7.2.0",
     "eslint-config-standard": "^16.0.2",
     "eslint-plugin-prettier": "^3.3.0",
     "husky": "^5.0.6",
     "lint-staged": "^10.5.3",
     "pinst": "^2.1.1",
     "prettier": "^2.2.1",
-    "rollup": "^2.36.0",
+    "rollup": "^2.36.2",
     "rollup-plugin-terser": "^7.0.2",
     "rollup-plugin-typescript2": "^0.29.0",
     "typescript": "^4.1.0",
-    "vite": "v2.0.0-beta.10",
+    "vite": "v2.0.0-beta.30",
     "vue": "^3.0.0",
```

Comment:
this is not only a dev dependency?

Reply:
yes it is ?

Reply:
actually, we don't even need vite here

Reply:
how can `vue` be a dev dependency if you import it?

please don;t resolve conversations when answered, only when actually  resolved.

Reply:
moved to peerDependencies in 3ffb1af